src/UberStrok.Realtime.Server.Game/DeathMatchGameRoom.cs

```diff
@@ -0,0 +1,83 @@
+﻿using log4net;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using UberStrok.Core.Common;
+using UberStrok.Core.Views;
+
+namespace UberStrok.Realtime.Server.Game
+{
+    class DeathMatchGameRoom : BaseGameRoom
+    {
+        private readonly static ILog s_log = LogManager.GetLogger(nameof(DeathMatchGameRoom));
+        private GameRoomDataView gameRoomDataView;
```

Comment:
`BaseGameRoom` already maintains a reference to the view instance under `BaseGameRoom.View`, so you can use that instead.